src/pages/tutorials/android-wallet.mdx

```diff
@@ -1083,8 +1102,62 @@ fun WriteContractView(viewModel: MainViewModel) {
 }
 ```
 
+## Advanced Features
+
+### Smart Accounts (ERC-4337)
+
+Web3Auth Android SDK v10.x includes built-in support for smart accounts. When enabled in your dashboard, users automatically get smart contract wallets with advanced features:
+
+```kotlin
+// Check if smart accounts are enabled
+val userInfo = web3AuthHelper.getUserInfo()
+if (userInfo.isSmartAccountEnabled) {
+    val smartAccountAddress = userInfo.smartAccountAddress
+    Log.d("Smart Account", "Address: $smartAccountAddress")
+
+    // Use smart account for gasless transactions
+    val userOp = buildUserOperation(smartAccountAddress, transaction)
+    web3AuthHelper.request("eth_sendUserOperation", userOp)
+}
```

Comment:
This whole section is not correct, and using Web3Auth Wallet Services as well. Please check the example, also there's no function/method such as buildUserOperation.